sdk/eventgrid/azure-eventgrid-system-events/README.md

```diff
@@ -0,0 +1,50 @@
+
+
+# Azure Event Grid System Events client library for Python
+<!-- write necessary description of service -->
+
+## Getting started
+
+### Install the package
+
+```bash
+python -m pip install azure-eventgrid-system-events
+```
+
+#### Prequisites
+
+- Python 3.8 or later is required to use this package.
+- You need an [Azure subscription][azure_sub] to use this package.
+- An existing Azure Event Grid System Events instance.
+
+## Key concepts
+## Examples
+## Troubleshooting
+## Next steps
```

Comment:
if these sections arent going to be filled lets remove them for now :)

Reply:
I didnt have them earlier but the readme check wants them